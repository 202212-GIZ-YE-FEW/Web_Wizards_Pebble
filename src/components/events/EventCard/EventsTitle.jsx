const EventsTitle = ({ title }) => {
    const shortTitle = title?.slice(0, 30) + "...";
    const longTitle = title?.slice(0, 40) + "...";

    return (
        <h2 className='card-title font-bold py-1 text-lg '>
            <span className='hidden sm:inline'>{longTitle}</span>
            <span className='inline sm:hidden'>{shortTitle}</span>
        </h2>
    );
};

export default EventsTitle;
