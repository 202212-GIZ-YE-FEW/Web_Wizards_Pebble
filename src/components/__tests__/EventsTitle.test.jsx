import renderer from "react-test-renderer";

import EventsTitle from "../events/EventCard/EventsTitle";

it("renders correctly", () => {
    const tree = renderer.create(<EventsTitle />).toJSON();
    expect(tree).toMatchSnapshot();
});
