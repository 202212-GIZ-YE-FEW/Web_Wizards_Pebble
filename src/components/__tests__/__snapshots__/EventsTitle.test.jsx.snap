// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2
  className="card-title font-bold py-1 text-lg "
>
  <span
    className="hidden sm:inline"
  >
    undefined...
  </span>
  <span
    className="inline sm:hidden"
  >
    undefined...
  </span>
</h2>
`;
